scene/resources/importer_mesh.cpp

```diff
@@ -1086,16 +1091,25 @@ Error ImporterMesh::lightmap_unwrap_cached(const Transform3D &p_base_transform,
 
 		} else {
 			for (int j = 0; j < ic / 3; j++) {
+				ERR_FAIL_INDEX_V(j * 3 + 0, rindices.size(), ERR_UNAVAILABLE);
+				ERR_FAIL_INDEX_V(rindices[j * 3 + 0], rvertices.size(), ERR_UNAVAILABLE);
 				Vector3 p0 = transform.xform(rvertices[rindices[j * 3 + 0]]);
+				ERR_FAIL_INDEX_V(j * 3 + 1, rindices.size(), ERR_UNAVAILABLE);
+				ERR_FAIL_INDEX_V(rindices[j * 3 + 1], rvertices.size(), ERR_UNAVAILABLE);
 				Vector3 p1 = transform.xform(rvertices[rindices[j * 3 + 1]]);
+				ERR_FAIL_INDEX_V(j * 3 + 2, rindices.size(), ERR_UNAVAILABLE);
+				ERR_FAIL_INDEX_V(rindices[j * 3 + 2], rvertices.size(), ERR_UNAVAILABLE);
```

Comment:
`ERR_UNAVAILABLE` is weird, that's for features which are not available, not for bad data.

The checks are also a bit overkill:
```suggestion
				ERR_FAIL_INDEX_V(j * 3 + 2, rindices.size(), ERR_UNAVAILABLE);
				ERR_FAIL_INDEX_V(rindices[j * 3 + 2], rvertices.size(), ERR_UNAVAILABLE);
				Vector3 p0 = transform.xform(rvertices[rindices[j * 3 + 0]]);
				Vector3 p1 = transform.xform(rvertices[rindices[j * 3 + 1]]);
```

Reply:
Changed to ERR_INVALID_DATA since invalid indices indicate a problem with the mesh.

I also removed the `ERR_FAIL_INDEX_V(j * 3 + 2, rindices.size(), ERR_UNAVAILABLE);` checks because it is checking the loop counter against the end index for the loop which is redundant `int ic = rindices.size()`

I had to keep all 3 of the `ERR_FAIL_INDEX_V(rindices[j * 3 + 2], rvertices.size(), ERR_INVALID_DATA);` checks because each individual vertex index might be wrong.